routers/web/repo/pull.go

```diff
@@ -1379,7 +1410,12 @@ func deleteBranch(ctx *context.Context, pr *issues_model.PullRequest, gitRepo *g
 		log.Error("DeleteBranch: %v", err)
 	}
 
-	ctx.Flash.Success(ctx.Tr("repo.branch.deletion_success", fullBranchName))
+	successMsg := ctx.Tr("repo.branch.deletion_success", fullBranchName)
+	if ctx.Flash.SuccessMsg != "" {
```

Comment:
Why we need this change?

Reply:
Because we can have a  adding a tag at the same time as deleting the branch. We should output both messages if both are selected.

The `if form.DeleteBranchAfterMerge` on line 1095 which calls this `deleteBranch` method is run after the `if form.TagAfterMerge` on line 1065